### ISSUE TITLE
NetFPGA Compatible Motherboards 

### ISSUE THREAD

**Issue by mharris-t** [NetFPGA/P4-NetFPGA-public#63]
Hi,

I have been looking into compatible motherboards for the NetFPGA NIC. I have tried various different motherboards supporting PCIe 3.0 slots. But none of them worked. Procuring motherboards from this [list](https://github.com/NetFPGA/NetFPGA-SUME-public/wiki/Motherboard-Information) has been quite difficult. I would like to know if anyone has been able to make the NIC work in some commonly available motherboards.

**Comment by salvatorg** [comment]
Have you tried to set in BIOS the PCIe to support Gen. 2 instead of 3?

`The RIFFA DMA Module is based on the DMA created by the RIFFA project (but is not identical). The module supports PCI Express Gen.2 x8`

**Comment by mharris-t** [comment]
@salvatorg 
> Have you tried to set in BIOS the PCIe to support Gen. 2 instead of 3?
> 
> 
> 
> `The RIFFA DMA Module is based on the DMA created by the RIFFA project (but is not identical). The module supports PCI Express Gen.2 x8`

Yes, I have done so. But my motherboard does not detect the NetFPGA even after loading the bitfile after successful compilation. I have tried several motherboards that we have at our disposal. But none of them could detect it or its interfaces when _lspci_ or _ifconfig_ commands are used in the terminal.

**Comment by lucasbrasilino** [comment]
@mharris-t 
Just to make sure, are you rebooting your machine without powering down the board (warm boot) after loading the bitfile ?

**Comment by mharris-t** [comment]
@lucasbrasilino 
> @mharris-t 
> 
> Just to make sure, are you rebooting your machine without powering down the board (warm boot) after loading the bitfile ?

Yes, I have rebooted several times with several tries. Basically, when I push the bitfile with program_switch.sh, I get the "interfaces not detected" message. I have reloaded and reinstalled the sume drivers several times too.

**Comment by mharris-t** [comment]
Please refer to the following issue I have opened a while ago.

https://github.com/NetFPGA/P4-NetFPGA-public/issues/62
